string coord
time timestamp
VelodynePacket[] packets
